go-controller/pkg/node/gateway_iptables.go

```diff
@@ -377,6 +382,12 @@ func initExternalBridgeServiceForwardingRules(cidrs []*net.IPNet) error {
 	return insertIptRules(getGatewayForwardRules(cidrs))
 }
 
+// delExternalBridgeServiceForwardingRules removes iptables rules which might
+// have been added to disable forwarding
+func delExternalBridgeServiceForwardingRules(cidrs []*net.IPNet) error {
+	return deleteIptRules(getGatewayForwardRules(cidrs))
```

Comment:
do we need the extra abstraction here? can't we just call `nodeipt.DelRules(getGatewayForwardRules(cidrs))` straight from here? its straightforward enough for readability that I don't see the need for abstracting

Reply:
I have done this to keep consistency with the initExternalBridgeServiceForwardingRules() and initExternalBridgeDropForwardingRules() function. init functions uses this abstraction mostly due to use of insertIptRules() in one and appendIptRules() in the other init function.

Reply:
@arghosh93 ^